src/adv_diff/AdvDiffSemiImplicitHierarchyIntegrator.cpp

```diff
@@ -908,23 +908,37 @@ AdvDiffSemiImplicitHierarchyIntegrator::postprocessIntegrateHierarchy(const doub
 /////////////////////////////// PROTECTED ////////////////////////////////////
 
 void
-AdvDiffSemiImplicitHierarchyIntegrator::resetHierarchyConfigurationSpecialized(
-    const Pointer<BasePatchHierarchy<NDIM> > base_hierarchy,
-    const int coarsest_level,
-    const int finest_level)
+AdvDiffSemiImplicitHierarchyIntegrator::regridHierarchyBeginSpecialized()
```

Comment:
Can you remove this specialization since it only calls the parent class implementation?

Reply:
Whoops, I think I misread this diff --- it is doing more than calling the parent class implementation.